src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -740,7 +790,10 @@
            hostprivkey)))
 
     :else
-    (generate-master-ssl-files! settings certname ca-settings)))
+    (generate-master-ssl-files! settings certname ca-settings))
+  ;; Generate a file containing serial numbers of the configured compiler master certificates, if any.
```

Comment:
I don't think you need this comment line

Reply:
Yeah. regret all the nits.